src/main/resources/subject-contact-details/translation.json

```diff
@@ -41,15 +41,15 @@
         },
         "cy": {
             "subjectEmailAddress": {
-                "required": "Please enter a valid email address",
-                "invalid": "Every email address consists of 3 elements: local-part, @ symbol (pronounced as “at”), and a domain name (Welsh)"
+                "required": "Nodwch gyfeiriad e-bost dilys",
+                "invalid": "Mae pob cyfeiriad e-bost yn cynnwys tair elfen: rhan lleol, symbol @ (a ynganir fel “at”), ac enw parth"
```

Comment:
interesting. don't we need to translate the words symbol and at?

Reply:
symbol in Welsh is the same as in English funnily enough